app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  validates :text, presence: true
```

Comment:
The comment model must have a method that updates the comments counter for a post as it's part of the requirement. Kindly implement that 👍